### PR TITLE
Add conflict with old webstatus version and release package [REVPI-1688 & REVPI-3118]

### PR DESCRIPTION
**Pull request by RamiGspo**
The revpi-config file has been imported from webstatus in commit 23b5ee5.
revpi-tools conflicts and will not operate if a old version of webstatus is installed on the system.
revpi-config file provided by older webstatus versions are not overwritten.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -33,6 +33,7 @@ Depends:
   ${shlibs:Depends}
 Suggests: python3-revpi-device-info, avahi-daemon
 Breaks: piserial (<< 3.0.0)
+Conflicts: revpi-webstatus (<< 1:2.1.0)
```
Comment (iluminat23) [inline debian/control:36]: I think this should be Breaks.
https://www.debian.org/doc/debian-policy/ch-relationships.html#packages-which-break-other-packages-breaks
https://www.debian.org/doc/debian-policy/ch-relationships.html#conflicting-binary-packages-conflicts
https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-and-replacing-packages-replaces

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,37 @@
+revpi-tools (2.0.1-1+revpi11+1) bullseye; urgency=medium
```
Comment (iluminat23) [inline debian/changelog:1]: How is this a 3.0.0-1 release?

Comment (RamiGspo) [reply debian/changelog]: I wanted to do this fast before our planning... :-1: I will avoid this again.

The tag is now in my Repo -> [tag v3.0.0](https://github.com/RamiGspo/revpi-tools/releases/tag/v3.0.0)

Comment (naruxde) [reply debian/changelog]: Sorry @RamiGspo, I told you, let's start the version 3.0.0 at that point. For version 3.0.0, two other PullRequests are missing, which must first be merged before the main branch is 3.0.0. Let's wait for it and then drive up with this PR.

**Review by naruxde — CHANGES_REQUESTED** [review]
Wait for version 3.0.0 of Branch master, or do a merge with master instead of v3.0.0.

**Comment by nbuchwitz** [comment]
> Wait for version 3.0.0 of Branch master, or do a merge with master instead of v3.0.0.

Wait for the other PRs 📃, re-create the tag 🏷️ and then merge ✍️ the tag into raspios branch (we don't want to merge the bare master) 🙏

**Review by nbuchwitz — DISMISSED** [review]

**Review by nbuchwitz — APPROVED** [review]

**Review by nbuchwitz — DISMISSED** [review]

**Comment by iluminat23** [comment]
Why adding REVPI-3118? This is not planed for now and other changes to the webstatus package are pending!

**Review by naruxde — APPROVED** [review]

**Review by iluminat23 — APPROVED** [review]

**Pull request merged**